ansible/roles/iroha-docker/defaults/main.yml

```diff
@@ -18,27 +18,24 @@ iroha_forced_cleaning: False
 # Default: false
 iroha_custom_hostnames: false
```

Comment:
Description for this variable is outdated because the "example" from `playbooks/iroha-docker/main.yml` was removed.

Reply:
fixed